truss/templates/shared/middleware.py

```diff
@@ -0,0 +1,79 @@
+from msgpack_asgi import MessagePackMiddleware
+from shared.serialization import truss_msgpack_deserialize, truss_msgpack_serialize
+from starlette.middleware.base import BaseHTTPMiddleware, RequestResponseEndpoint
+from starlette.requests import Request
+from starlette.responses import Response
+from starlette.types import ASGIApp
+
+OCTET_STREAM_CONTENT_TYPE = "application/octet-stream"
+MSGPACK_CONTENT_TYPE = b"application/x-msgpack"
+
+
+class BinaryHeaderMiddleware(BaseHTTPMiddleware):
```

Comment:
This seems to be simply mapping OCTET_STREAM_CONTENT_TYPE to MSGPACK_CONTENT_TYPE. Should we rename this class to something like BinaryToMessagePackContentTypeHeaderMiddleware to make this obvious?